resources/js/components/felte-form/FormDebug.tsx

```diff
@@ -0,0 +1,20 @@
+import {VoidComponent} from "solid-js";
+import {useFormContext} from "./FelteForm";
+
+/** A component rendering debugging information about a form. Not for production. */
+export const FormDebug: VoidComponent = () => {
```

Comment:
Could we somehow check for env != dev and display nothing in such case?

Reply:
Mamy tę stałą DEV której używamy do wykrywania nie tyle środowiska testowego (to nie wiem jak łatwo zrobić) co trybu budowania DEV, to będzie spoko. No ale chyba nadal raczej staramy się nie wsubmitować żadnego użycia tego w kodzie.

Dobra, zrobiłem jeszcze warninga lintowego.